routes/ProductionRequest.ts

```diff
@@ -0,0 +1,121 @@
+import { Request, Response, NextFunction } from 'express';
+import Joi from '@hapi/joi';
+import logger from '../config/logger';
+import GovDeliveryService from '../services/GovDeliveryService';
+import { DevPortalError } from '../models/DevPortalError';
+import {validateApiList} from './DeveloperApplication';
+
+export const productionSchema = Joi.object().keys({
+  primaryContact: Joi.object({
+    firstName: Joi.string().required(),
+    lastName: Joi.string().required(),
+    email: Joi.string().required(),
+  }).required(),
+  secondaryContact: Joi.object({
+    firstName: Joi.string().required(),
+    lastName: Joi.string().required(),
+    email: Joi.string().required(),
+  }).required(),
+  organization: Joi.string().required(),
+  appName: Joi.string().required(),
+  appDescription: Joi.string().required(),
+  statusUpdateEmails: Joi.array().items(Joi.string()).required(),
+  valueProvided: Joi.string().required(),
+  businessModel: Joi.string(),
+  policyDocuments: Joi.array().items(Joi.string()).required(),
+  phoneNumber: Joi.string().required(),
+  apis: Joi.custom(validateApiList).required(),
+  monitizedVeteranInformation: Joi.boolean().required(),
+  monitizationExplanation: Joi.string(),
+  veteranFacing: Joi.boolean(),
+  website: Joi.string(),
+  signUpLink: Joi.string(),
+  supportLink: Joi.string(),
+  platforms: Joi.array().items(Joi.string()),
+  veteranFacingDescription: Joi.string(),
+  vasiSystemName: Joi.string(),
+  credentialStorage: Joi.string().required(),
+  storePIIOrPHI: Joi.boolean().required(),
+  storageMethod: Joi.string(),
+  safeguards: Joi.string(),
+  breachManagementProcess: Joi.string(),
+  vulnerabilityManagement: Joi.string(),
+  exposeHealthInformationToThirdParties: Joi.boolean(),
+  thirdPartyHealthInfoDescription: Joi.string(),
+  scopesAccessRequested: Joi.array().items(Joi.string()),
+  distrubitingAPIKeysToCustomers: Joi.boolean(),
+  namingConvention: Joi.string(),
+  centralizedBackendLog: Joi.string(),
+  listedOnMyHealthApplication: Joi.boolean(),
+}).options({ abortEarly: false });
```

Comment:
This is one of the main issues with the image uploads and weren't included in the schema since I wasn't entirely sure what route we were going to be doing for images. GovDelivery doesn't currently allow for attachments, so we'd need to either refactor govdelivery to allow for image attachments + add the files to be added to the schema OR pass in image source urls as strings into the schema. 
The other thing we found the other day is that gmail doesn't render base64 images so I'm sure that's an issue we'd run into in other clients as well. 
The other week when talking to Brent he had given me the impression that we were going to be handling all images as source URLs for the MVP, but I'll need to double check that. To that point there still needs to be fields for the `appImage`, `patientWaitTime`, and `medicalDisclaimer` images, regardless.